src/heroic_series.ts

```diff
@@ -221,27 +221,32 @@ export default class HeroicSeries {
     return table;
   }
 
-  public buildScoped(group, common) {
-    let scoped = {};
-    for (let tk in group.tagCounts) {
-      scoped[`tag_${tk}`] = { text: "<" + group.tagCounts[tk] + ">" };
-      scoped[`{tag_${tk}_count`] = { text: "<" + group.tagCounts[tk] + ">" };
+  public buildScopedHelper(scoped, counts, tags, common) {
+    for (let tk in counts) {
```

Comment:
You could do `for(const [tk, tv] of Object.entries(counts))` here. I think it has better performance but I'm not certain about that.

Reply:
Point taken, but going to ignore for now since typescript does not support Object.entries and thus will require a different approach